From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 11 Mar 2021 23:20:05 +0000
Subject: [PATCH] Fix expected result for a setfattr test

It appears that `setfattr` is able to set extended attributes for a
symlink, but upstream expects that operation to fail...

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 test/attr.test | 1 -
 1 file changed, 1 deletion(-)

diff --git a/test/attr.test b/test/attr.test
index 6ce2f9b..11a3413 100644
--- a/test/attr.test
+++ b/test/attr.test
@@ -205,7 +205,6 @@ Attributes of symlinks vs. the files pointed to
 	$ setfattr -n user.filename -v f l
 	$ setfattr -n user.filename -v l l
 	$ setfattr -h -n user.filename -v l l
-	> setfattr: l: Operation not permitted
 	
 	$ getfattr -d f l | sort-getfattr-output
 	> # file: f
